### PR TITLE
Fix iOS build status in CircleCI

### PR DESCRIPTION
**Pull request by matteblair**

### CODE REVIEW

**Comment by matteblair** [comment]
☝️That little green checkmark next to the last commit is the problem I'm trying to fix. The iOS build is clearly broken, but the CircleCI script doesn't detect the failure.

I suspect it's because we're piping Xcode's output through xcpretty. This has been an issue before, though I thought I'd fixed it the last time I changed this part of the makefile: https://github.com/tangrams/tangram-es/commit/32b95786dc6e6e2c96ff21f61262f7d1734100fb

**Comment by matteblair** [comment]
☝️ The failing status for this commit means that the CI script is now correctly detecting failures. Now I can drop the previous commit.

**Comment by matteblair** [comment]
As expected - the failing iOS builds are now being detected as CI failures.

And now, for my last feat, I will vanish the errors entirely!

File: platforms/ios/demo/TangramDemo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -411,7 +415,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = G436CJ835H;
```
Comment (tallytalwar) [inline platforms/ios/demo/TangramDemo.xcodeproj/project.pbxproj]: oops, I would prefer dev team string empty here. I know xcode added it automatically.

Comment (matteblair) [reply platforms/ios/demo/TangramDemo.xcodeproj/project.pbxproj]: Gah! Pay no attention to the DEVELOPMENT_TEAM behind the curtain!

**Pull request merged**